extensions/amp-lightbox-gallery/1.0/component.jss.js

```diff
@@ -52,6 +52,36 @@ const lightbox = {
   },
 };
 
+const grid = {
+  display: 'grid !important',
+  justifyContent: 'center !important',
+  gridGap: '5px !important',
+  gridTemplateColumns: 'repeat(3, 1fr)',
+  gridAutoRows: 'min-content !important',
+  padding: '0px 5px !important',
+  top: '56px !important',
+  height: 'calc(100% - 56px) !important',
+  width: 'calc(100% - 10px) !important',
+  '@media (min-width: 1024px)': {
+    gridTemplateColumns: 'repeat(4, calc(1024px/4 - 5px * 5 / 4))',
+    top: '80px !important' /* Matches height of top-bar */,
```

Comment:
Define `topBarHeight` instead? It might be useful to go through this file similarly.

```suggestion
    top: `${topBarHeight}px !important`,
```

Reply:
Done, thanks for the rec!